modules/create-vm/pkg/vmcreator/vm-creator.go

```diff
@@ -15,7 +15,7 @@ import (
 	"github.com/kubevirt/kubevirt-tekton-tasks/modules/shared/pkg/zerrors"
 	"github.com/kubevirt/kubevirt-tekton-tasks/modules/shared/pkg/zutils"
 	templatev1 "github.com/openshift/client-go/template/clientset/versioned/typed/template/v1"
-	k8sv1 "k8s.io/client-go/kubernetes/typed/core/v1"
+	k8sv1 "k8s.io/client-go/kubernetes"
```

Comment:
Unusual import name for the client, in other files you are not changing the import name of the client.